### DISCUSSION TITLE
[bug] istallation problem in xamp

### DISCUSSION THREAD

**Discussion by fatmazayedsayed** [YetiForceCompany/YetiForceCRM#16837] [Ignored/Incomplete]
i am using windows + xamp to install yetforce
steps 
i downloaded YetiForceCRM-6.4.0-complete.zip
then extract it in xamp/htdocs

i have the following error 
Fatal error: Uncaught Exception: The intl library is missing in D:\xampp\htdocs\yetiforce\include\RequirementsValidation.php:15 Stack trace: #0 D:\xampp\htdocs\yetiforce\install\Install.php(23): include_once() #1 {main} thrown in D:\xampp\htdocs\yetiforce\include\RequirementsValidation.php on line 15

**Comment by fatmazayedsayed** [comment]
for me open php.ini
then update the last part it to be (remove some ;)



extension=bz2
extension=curl
;extension=ffi
;extension=ftp
extension=fileinfo
extension=gd
extension=gettext
extension=gmp
extension=intl
extension=imap
extension=ldap
extension=mbstring
extension=exif      ; Must be after mbstring as it depends on it
extension=mysqli
;extension=oci8_12c  ; Use with Oracle Database 12c Instant Client
;extension=oci8_19  ; Use with Oracle Database 19 Instant Client
extension=odbc
;extension=openssl
;extension=pdo_firebird
extension=pdo_mysql
;extension=pdo_oci
;extension=pdo_odbc
;extension=pdo_pgsql
extension=pdo_sqlite
;extension=pgsql
;extension=shmop

**Comment by mariuszkrzaczkowski** [comment]
bug/error >> https://github.com/YetiForceCompany/YetiForceCRM/issues/new?assignees=&labels=&template=1-bug-report.md&title=%5Bbug%5D+
question >> https://github.com/YetiForceCompany/YetiForceCRM/discussions

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/issues/13908
